File: testprojects/scala_main/pipelines/LIVY_SCALA/code/src/main/scala/io/prophecy/pipelines/livy_scala/graph/livyscalaSG1_1/OrderBy_1_1.scala
package io.prophecy.pipelines.livy_scala.graph.livyscalaSG1_1

import io.prophecy.libs._
import io.prophecy.pipelines.livy_scala.config.ConfigStore._
import io.prophecy.pipelines.livy_scala.udfs.UDFs._
import io.prophecy.pipelines.livy_scala.udfs._
import org.apache.spark._
import org.apache.spark.sql._
import org.apache.spark.sql.functions._
import org.apache.spark.sql.types._
import org.apache.spark.sql.expressions._
import java.time._

object OrderBy_1_1 {

  def apply(spark: SparkSession, in: DataFrame): DataFrame =
    in.orderBy(
      col("year").asc,
      col("industry_code_ANZSIC").asc,
      col("industry_name_ANZSIC").asc,
      col("rme_size_grp").asc,
      col("variable").asc,
      col("value").asc,
      col("unit").asc
    )

}

File: testprojects/python_main/pipelines/LIVY_PYTHON/code/livy_python/graph/pythonLivySG1_1/OrderBy_1_1.py
from pyspark.sql import *
from pyspark.sql.functions import *
from pyspark.sql.types import *
from prophecy.libs import typed_lit
from .config import *
from livy_python.udfs.UDFs import *

def OrderBy_1_1(spark: SparkSession, in0: DataFrame) -> DataFrame:
    return in0.orderBy(
        col("year").asc(), 
        col("industry_code_ANZSIC").asc(), 
        col("industry_name_ANZSIC").asc(), 
        col("rme_size_grp").asc(), 
        col("variable").asc(), 
        col("value").asc(), 
        col("unit").asc()
    )
